Services/Sessions/WebAppSession.m

```diff
@@ -299,4 +299,29 @@ - (ServiceSubscription *)subscribeMediaInfoWithSuccess:(SuccessBlock)success fai
     return nil;
 }
 
+#pragma mark - Playlist Control
+
+- (id <PlayListControl>)playListControl
+{
+    return self;
+}
+
+- (void)playNextWithSuccess:(SuccessBlock)success failure:(FailureBlock)failure{
+    if (failure){
```

Comment:
Why don't you use `[self sendNotSupportedFailure:failure];`, as it's done above?